src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -25,8 +25,8 @@
     <div
       v-if="currentPeopleAndRoles.length > 0"
       id="people-roles-list"
+      class="section-container"
```

Comment:
Can you provide before and after screenshots in edit view and summary view, please?

Reply:
Before:

![image](https://user-images.githubusercontent.com/116035339/223827888-4f29c163-b4b4-47ab-8784-1b1c51cf7ea8.png)

After:

![image](https://user-images.githubusercontent.com/116035339/223827969-d342eab6-b45e-42a0-af35-a8948fc760d9.png)

Reply:
Please test this with a change filing (in which the org/person information is not in its own section).

For example:

![image](https://user-images.githubusercontent.com/10002889/223830453-3d236336-6c41-4ec5-91cc-d7970a0ca9d8.png)

Reply:
Good catch. I tried to test as much as possible but will test this scenario and make sure it works

Reply:
Instead of changing style in component, handling this in Restoration.vue